rpc/errors.go

```diff
@@ -108,7 +108,7 @@ var (
 	}
 	ErrBlockNotFound = &RPCError{
 		code:    24,
-		message: "Block not found",
+		message: "Invalid block hash",
```

Comment:
I don't think this message needed to be changed  https://github.com/starkware-libs/starknet-specs/blob/c0980ed9aa5a8eb2def8d9b145fba27b4e29b7f9/api/starknet_api_openrpc.json#L3345

Reply:
I looked a bit more into it. It appears there is an inconsistency on the spec. BLOCK_NOT_FOUND error is defined twice with different error messages: [here](https://github.com/starkware-libs/starknet-specs/blob/c0980ed9aa5a8eb2def8d9b145fba27b4e29b7f9/api/starknet_api_openrpc.json#L3345) and [here](https://github.com/starkware-libs/starknet-specs/blob/c0980ed9aa5a8eb2def8d9b145fba27b4e29b7f9/api/starknet_trace_api_openrpc.json#L483C14-L483C29). "Block not found" seems like the correct message.